### PR TITLE
Add cpu time to E2 data

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: lua/autorun/server/sv_cfc_contact_forms.lua [hunk 1]
```diff
@@ -62,6 +62,7 @@ local function getE2Information()
             local e2Info = {}
             e2Info.name = expression2:GetGateName()
             e2Info.size = expression2:GetCode():len()
+            e2Info.cpuTime = expression2.context.timebench
```
Comment (ghost) [inline lua/autorun/server/sv_cfc_contact_forms.lua]: The CPU time is provided in seconds. To get the CPU time in us ( Like it is displayed on the E2 ui ) you need to multiply it by 1000000. Should I do it here already?
```suggestion
            e2Info.cpuTime = expression2.context.timebench * 1000000
```

Comment (samuelWilliams99) [reply lua/autorun/server/sv_cfc_contact_forms.lua]: If CPU time is in seconds, then yeah this makes sense, might be a good call to round it to an int after though

**Comment by sarahsturgeon** [comment]
retest this please

**Comment by ghost** [comment]
retest this please

**Comment by sarahsturgeon** [comment]
This looks like it's already supported in here

**Pull request closed without merging**